platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultConfigurationCache.kt

```diff
@@ -102,43 +99,28 @@ class DefaultConfigurationCache internal constructor(
     var cacheEntryRequiresCommit = false
 
     private
-    val host by lazy { deferredRootBuildGradle.gradle.services.get<HostServiceProvider>() }
+    val host by lazy { host.instance }
```

Comment:
❓ What's the benefit of being `lazy` rather than just a property getter?

Reply:
None really, because the backing service wrapper ensures the memorization and synchronization.

I didn't see your comment, so I force-pushed in the meantime, and now it's even neater with the property delegation (that uses the just the getter under the hood)